apiary.apib

```diff
@@ -9621,3 +9629,12 @@ Fetch which version of Projects the account is using.
 + `friday`
 + `saturday`
 + `sunday`
+
+## TimeTrackingRelatesToTypes (enum)
++ `contact`
++ `company`
++ `project`
++ `milestone`
++ `ticket`
++ `nextgenProject`
++ `nextgenGroup`
```

Comment:
Q: I don't think we actually support time trackings related to nextgen groups, unless I missed something
/cc @adognameddog

Reply:
There is an open PR with the changes to include it

Reply:
What do you mean @gehadMohamed? Allowing time tracking on project groups is a massive feature. Can you point me to that open PR?